src/components/refactor/dialogs/voltage-level-creation/voltage-level-creation-dialog.js

```diff
@@ -0,0 +1,211 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { useSnackMessage } from '@gridsuite/commons-ui';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { sanitizeString } from 'components/dialogs/dialogUtils';
+import EquipmentSearchDialog from 'components/dialogs/equipment-search-dialog';
+import { useFormSearchCopy } from 'components/dialogs/form-search-copy-hook';
+import {
+    BUS_BAR_CONNECTIONS,
+    BUS_BAR_SECTIONS,
+    EQUIPMENT_ID,
+    EQUIPMENT_NAME,
+    FROM_BBS,
+    HORIZONTAL_POSITION,
+    ID,
+    NAME,
+    NOMINAL_VOLTAGE,
+    SUBSTATION_ID,
+    SWITCH_KIND,
+    TO_BBS,
+    VERTICAL_POSITION,
+} from 'components/refactor/utils/field-constants';
+import yup from 'components/refactor/utils/yup-config';
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+import { createVoltageLevel } from 'utils/rest-api';
+import ModificationDialog from 'components/refactor/dialogs/commons/modificationDialog';
+
+import VoltageLevelCreationForm from './voltage-level-creation-form';
+import {
+    controleBusBarSectionLink,
+    controleUniqueHorizontalVertical,
+    controleUniqueId,
+} from './voltage-level-creation-utils';
+
+/**
+ * Dialog to create a load in the network
+ * @param currentNode The node we are currently working on
+ * @param studyUuid the study we are currently working on
+ * @param editData the data to edit
+ * @param onCreateVoltageLevel to create voltage level from other forms,
+ * @param dialogProps props that are forwarded to the generic ModificationDialog component
+ */
+
+const emptyFormData = {
+    [EQUIPMENT_ID]: '',
+    [EQUIPMENT_NAME]: '',
+    [NOMINAL_VOLTAGE]: '',
+    [SUBSTATION_ID]: '',
```

Comment:
default value should be empty, and this field should be nullable to avoid wrong error message when field is emptied